redis/client.py

```diff
@@ -3281,6 +3287,22 @@ def zunionstore(self, dest, keys, aggregate=None):
         """
         return self._zaggregate('ZUNIONSTORE', dest, keys, aggregate)
 
+    def zmscore(self, key, members):
+        """
+        Returns the scores associated with the specified members
+        in the sorted set stored at key.
+
+        ``members`` should be a list of the member name.
+        Return type is a list of score.
+
+        If the member does not exist, a None will be returned
+        in corresponding position.
+        """
+        if not isinstance(members, list) or len(members) < 1:
```

Comment:
```suggestion
        if not members:
```

Reply:
So we should check if it's None or not and if it's a list type data?

Reply:
According to the [zmscore docs](https://redis.io/commands/zmscore) ZMSCORE requires a minimum of one member. This either means that we either:

1. Validate that there's a non-zero length list (as above).
2. Validate that the item passed in is a string or a dictionary.

I'd rather we not overload the members argument, and instead always pass in a list. For the simple case, the list has one item (i.e ['foo']). Validating that we were passed a list I think is overkill, as the docs and the function signature are indicative of the requirement.